config.json

```diff
@@ -91,7 +91,13 @@
       "core": false,
       "difficulty": 3,
       "slug": "raindrops",
-      "topics": null,
+      "topics": [
+        "strings",
+        "integers",
+        "loops",
+        "conditionals",
+        "mathematics"
```

Comment:
These look like great topic suggestions! `strings`, `integers`, `conditionals` and `mathematics` all definitely seem like topics associated with this exercise :)

I'm not sure about `loops` though. Since you're just checking three different factors using a loop might be overdoing it? So adding `loops` to the suggested topics might be encouraging people to use a non-ideal solution? What do you think? :)

Reply:
You're right. But the reason I included `loops` is because they might be used when getting the factors of the number. Other than that there wouldn't be any other ideal solutions that makes use of loops.

But maybe the `loops` tag can be covered by the `mathematics` tag instead.. In that case I can remove the `loops` tag.

Reply:
You're right, you could use loops to find the factors of a number. But for this problem you don't need to find all the factors of a number, you just need to check whether 3, 5, and 7 are factors. So I think adding `loops` as a topic might be a bit confusing to the people solving the exercise. What do you think? :)

Reply:
Yeah you're right. I'll change the tags accordingly.